person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_reader :name, :age, :parent_permission
+  attr_accessor :id
```

Comment:
- [ ] Please add name and age as `attr_accessor` because it's a getter and setter values.
- [ ] Kindly add id as `attr_reader` because it's getters only value.

![image](https://user-images.githubusercontent.com/68796232/173562985-275db53f-b040-4674-9dcc-5c1b729c6fac.png)